certs_test.go

```diff
@@ -0,0 +1,60 @@
+/*
+ * Minio Cloud Storage, (C) 2015, 2016 Minio, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package main
+
+import (
+	"os"
+	"path"
+	"strings"
+	"testing"
+)
+
+// Make sure we have a valid certs path.
+func TestGetCertsPath(t *testing.T) {
+	certsPath, err := getCertsPath()
+	if err != nil {
+		t.Error(err)
+	}
+	if certsPath == "" {
+		t.Errorf("expected path to not be an empty string, got: '%s'", certsPath)
+	}
+	// Ensure it contains some sort of path separator.
+	if !strings.ContainsRune(certsPath, os.PathSeparator) {
+		t.Errorf("expected path to contain file separator")
+	}
+	// It should also be an absolute path.
+	if !path.IsAbs(certsPath) {
```

Comment:
you have to use 'filepath.IsAbs()' since 'getCertsPath' returns a windows specific path.